### ISSUE TITLE
LineWrap stops working when lines are added to text that exceeds one page

### ISSUE THREAD

**Issue by synth3** [TomasMikula/RichTextFX#421]
I have found the following problem where line wrap stops working sometimes:

Reproduction:
- open the RichText demo
- turn off LineWrap
- Paste the text from the attached file [sampleContent.txt](https://github.com/TomasMikula/RichTextFX/files/696103/sampleContent.txt)
- add one linebreak at the end of the text content (by pressing ENTER)
- klick at the Wrap CheckBox to turn on LineWrapping again
Result:
LineWrap is not applied anymore

**Comment by JordanMartinez** [comment]
Yup. That's a bug! but the code seems to have been implemented correctly, so I think this might be a bug with JavaFX, not with RTFX

**Comment by JordanMartinez** [comment]
Even forcing the layout via `layout` does nothing.

**Comment by JordanMartinez** [comment]
I came across this issue again in #504 when I was implementing `NavigationTests.MuliLineTests`. I found one solution by hiding and then re-showing the stage.

**Comment by Jugen** [comment]
This only happens if the vertical scrollbar IS being displayed.
